mappings/net/minecraft/entity/mob/ShulkerEntity.mapping

```diff
@@ -3,34 +3,40 @@ CLASS net/minecraft/unmapped/C_wepsgefi net/minecraft/entity/mob/ShulkerEntity
 	FIELD f_eoeowyqr COVERED_ARMOR_BONUS_ID Ljava/util/UUID;
 	FIELD f_faxfphpy NO_COLOR B
 	FIELD f_fwpmtaxc MAX_TELEPORT_DISTANCE I
+	FIELD f_gvaaancy PEEK_CHANCE_PER_TICK F
 	FIELD f_hlyfzsza teleportLerpTimer I
 	FIELD f_hxfmsvrp TELEPORT_STEPS I
 	FIELD f_kdytjkmi ATTACHED_FACE Lnet/minecraft/unmapped/C_rinmcaxy;
 	FIELD f_nhqhcoaz OTHER_SHULKER_SCAN_RADIUS I
 	FIELD f_odaehnvp prevAttachedBlock Lnet/minecraft/unmapped/C_hynzadkk;
 	FIELD f_qyhdlubn PEEK_AMOUNT Lnet/minecraft/unmapped/C_rinmcaxy;
+	FIELD f_rhlsurhw FORWARD Lnet/minecraft/unmapped/C_zviwfwgf;
 	FIELD f_sxhxgvpw DEFAULT_COLOR B
 	FIELD f_tdqqxmxs openProgress F
 	FIELD f_twtbkvow COLOR Lnet/minecraft/unmapped/C_rinmcaxy;
 	FIELD f_yecffwtx COVERED_ARMOR_BONUS Lnet/minecraft/unmapped/C_hdbqsqsm;
 	FIELD f_yejiwmfa OTHER_SHULKER_LIMIT I
+	FIELD f_zdllrwfz MAX_LID_OPEN F
 	METHOD m_afxezjrh setColor (Lnet/minecraft/unmapped/C_arllgqae;)V
 		ARG 1 color
 	METHOD m_agmrwfit createShulkerAttributes ()Lnet/minecraft/unmapped/C_sdjuuzrz$C_tehwrjus;
-	METHOD m_aipehzdv (Lnet/minecraft/unmapped/C_xpuuihxf;FF)Lnet/minecraft/unmapped/C_hbcjzgoe;
+	METHOD m_aipehzdv getProgressDeltaShape (Lnet/minecraft/unmapped/C_xpuuihxf;FF)Lnet/minecraft/unmapped/C_hbcjzgoe;
 		ARG 0 direction
 	METHOD m_bcbqzxod getAttachedFace ()Lnet/minecraft/unmapped/C_xpuuihxf;
 	METHOD m_bzwsabfi spawnNewShulker ()V
 	METHOD m_cuftthil findAttachSide (Lnet/minecraft/unmapped/C_hynzadkk;)Lnet/minecraft/unmapped/C_xpuuihxf;
 		ARG 1 pos
-	METHOD m_hqsastbf (Lnet/minecraft/unmapped/C_xpuuihxf;F)Lnet/minecraft/unmapped/C_hbcjzgoe;
+	METHOD m_ghizitzm getPhysicalPeak (F)F
+		ARG 0 openProgress
+	METHOD m_hqsastbf getProgressShape (Lnet/minecraft/unmapped/C_xpuuihxf;F)Lnet/minecraft/unmapped/C_hbcjzgoe;
```

Comment:
This sounds weird

Reply:
It's getting the bounding box when the shulker is opeing itself. Not sure how else to name it

Reply:
How about `getOpeningBoundingBox`?